prac_06/guitar.py

```diff
@@ -0,0 +1,31 @@
+"""CP1404 Practical - Guitars!
+Estimated time: 40 minutes
+Actual time: 50 minutes """
+
+YEAR = 2023
+VINTAGE_THRESHOLD = 50
+
+
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        """
+        Initialise a Programming language instance.
```

Comment:
Minor typo here - you clearly mean 'Guitar' instance :-)

Reply:
Thanks, fixed